AnimationCodelab/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    ext {
+        compose_version = '1.0.0-SNAPSHOT'
+    }
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:7.0.0-alpha06'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.30'
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        maven { url 'https://androidx.dev/snapshots/builds/7141639/artifacts/repository' }
+    }
+}
+
+subprojects {
```

Comment:
This shouldn't be needed?

Reply:
Done.
Using "androidx.activity:activity-compose:1.3.0-SNAPSHOT" seems to make this unnecessary.